collections/_article/godotsharp-packages-net8.md

```diff
@@ -0,0 +1,68 @@
+
+<figure>
+	<picture>
+		<source srcset="/storage/blog/dotnet/release-schedule-dark.svg" media="(prefers-color-scheme: dark)">
+		<img class="lightbox-ignore" style="background-color: transparent;" src="/storage/blog/dotnet/release-schedule-light.svg">
+	</picture>
+	<figcaption><small>.NET release cadence from the <a href="https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core">.NET official support policy</a>.</small></figcaption>
+</figure>
+
+## What does this mean for users?
+
+This means your project now needs to target .NET 8 or newer, otherwise it will be incompatible with the new `GodotSharp` packages. But you can still use libraries that target older .NET versions.
+
+Existing projects will be automatically upgraded to target `net8.0` when they are opened with Godot 4.4. If your project already targets .NET 8 or newer, nothing will change. Thanks to [Andreia Gaita](https://github.com/shana) for implementing this in [GH-82927](https://github.com/godotengine/godot/pull/82927).
+
+## What took so long?
+
+The version of .NET that a library targets becomes the minimum required version for consumers of that library. That is, if the `GodotSharp` package targets `net7.0`, your game project needs to target `net7.0` or newer. We didn't want to force our users to upgrade to a non-LTS release.
+
+When .NET 8 was released, it would've been a good time to bump the target version in Godot C# packages, since .NET 8 is an LTS release. Unfortunately, there were some breaking changes in .NET 7[^1] that prevented us from upgrading and making the required changes took us longer than we hoped.
+
+However, users could always use the latest version of .NET in their projects. Up until now, .NET 6 has been the minimum required version, but newer versions have always been supported from day one (even the pre-releases[^2]). In 4.4, the new minimum is now .NET 8, but as always, your projects can target the recent STS release .NET 9 and any newer version that is released in the future.
+
+Libraries tend to stay on older versions of .NET to support as many consumers as possible. But upgrading to the latest version of .NET brings benefits that we want to take advantage of.
+
+[^1]: .NET 7 introduced the new `scoped` keyword. This feature breaks compatibility in some low-level scenarios that affected Godot.
+
+[^2]: While pre-releases are usually supported by Godot, they may require users to enable them by setting the `DOTNET_ROLL_FORWARD_TO_PRERELEASE` environment variable to `1`. Keep in mind pre-releases are still a work-in-progress so there may be bugs.
+
+## What about older Godot releases?
+
+We encourage users to upgrade to the latest version of Godot. We work hard to prevent breaking compatibility, to ensure you can safely upgrade to the next version of Godot when it's released. Please, make sure to test the pre-releases and let us know if you find some bugs so we can fix them before the stable release.
```

Comment:
```suggestion
We encourage users to upgrade to the latest version of Godot. We work hard to prevent breaking compatibility, to ensure you can safely upgrade to new versions of Godot when they are released. Please, make sure to test the pre-releases and let us know if you find some bugs so we can fix them before the 4.4 stable release.
```

Reply:
I intended to keep this version-agnostic, since testing pre-releases and reporting bugs still applies beyond 4.4. But I think it sounds good either way.

Reply:
No probs, removed "4.4".